api/android/README.md

```diff
@@ -151,10 +160,11 @@ $ bash ./api/android/build-android-lib.sh
 
 After building the Android API, you can find the library(.aar) in `$NNSTREAMER_ROOT/android_lib`.
 - Build result
-  1. nnstreamer.aar: NNStreamer library
-  2. nnstreamer-native.zip: shared objects and header files for native developer
+  1. nnstreamer-[BUILD_DATE].aar: NNStreamer library for Android
+  2. nnstreamer-native-[BUILD_DATE].zip: shared objects and header files for native developer
+  3. nnstreamer-static-libs-[BUILD_DATE].zip: static library files (i.e. .a)
```

Comment:
Actually static libs are unnecessary for app developers (just for the license)
and I updated mk files to build shared library now.

Reply:
I agreed.